docker/msfconsole.rc

```diff
@@ -6,5 +6,5 @@ else
   lhost = %x(hostname -i)
 end
 run_single("setg LHOST #{lhost}")
-run_single("db_connect #{ENV['DATABASE_URL'].gsub('postrgres://', '')}") if ENV['DATABASE_URL']
```

Comment:
This is needed because of https://github.com/rapid7/metasploit-framework/blob/master/docker-compose.yml#L9 and https://github.com/rapid7/metasploit-framework/blob/master/docker/docker-compose.development.override.yml#L10
So either leave the statement or also change the yml files